lib/mix/test/mix/tasks/profile.fprof_test.exs

```diff
@@ -11,38 +11,44 @@ defmodule Mix.Tasks.Profile.FprofTest do
   test "profiles evaluated expression", context do
     in_tmp context.test, fn ->
       assert capture_io(fn ->
-        Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)"])
-      end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
+               Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)"])
+             end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
     end
   end
 
   test "profiles the script", context do
     in_tmp context.test, fn ->
       profile_script_name = "profile_script.ex"
 
-      File.write! profile_script_name, """
+      File.write!(profile_script_name, """
       Enum.each(1..5, fn(_) -> MapSet.new end)
-      """
+      """)
 
       assert capture_io(fn ->
-        Fprof.run([profile_script_name])
-      end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
+               Fprof.run([profile_script_name])
+             end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
     end
   end
 
   test "expands callers", context do
     in_tmp context.test, fn ->
       assert capture_io(fn ->
-        Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)", "--callers"])
-      end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3} +<--)
+               Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)", "--callers"])
+             end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3} +<--)
     end
   end
 
   test "expands processes", context do
     in_tmp context.test, fn ->
-      output = capture_io(fn ->
-        Fprof.run(["-e", "spawn(fn -> Process.sleep(:infinity) end); Enum.each(1..5, fn(_) -> MapSet.new end)", "--details"])
-      end)
+      output =
+        capture_io(fn ->
+          Fprof.run([
+            "-e",
+            "spawn(fn -> Process.sleep(:infinity) end); Enum.each(1..5, fn(_) -> MapSet.new end)",
```

Comment:
Can you please extract this out so everything is a single line?

```
expr = "spawn(fn -> Process.sleep(:infinity) end); Enum.each(1..5, fn(_) -> MapSet.new end)"
output = capture_io(fn -> Fprof.run(["-e", expr, "--details"]) end)
assert output =~ ...
```

Reply:
yes i will do that